engines/wintermute/wintermute.cpp

```diff
@@ -439,10 +439,8 @@ bool WintermuteEngine::getGameInfo(const Common::FSList &fslist, Common::String
 					}
 					caption = value;
 
-					for (uint i = 0; i< value.size(); i++)
```

Comment:
formatting is still not valid, also please use amend, not intentionally add new commit